scripts/Muon/GUI/Common/seq_fitting_tab_widget/SequentialTableModel.py

```diff
@@ -0,0 +1,182 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import (absolute_import, division, print_function)
+from qtpy.QtCore import QAbstractTableModel, Qt, Signal, QModelIndex
+
+default_table_columns = ["Run", "Group/Pairs", "Fit status", "Chi squared"]
```

Comment:
can we name this file something else please? Model suggests that it is do calculations and does not contain Qt. I think this looks like a base class for the view.